charts/k8s-monitoring/README.md

```diff
@@ -9,6 +9,12 @@ A Helm chart for gathering, scraping, and forwarding Kubernetes infrastructure m
 
 ## Breaking change announcements
 
+### **v0.12.0**
+
+  The component `loki.write.grafana_cloud_loki` has been renamed.
```

Comment:
I see just one stray reference to this in `Adding custom Flow configuration` of the README.